native-heapdump-example/README.md

```diff
@@ -0,0 +1,130 @@
+# Create a Heap Dump from a Native Executable
+
+You can create a heap dump of a running executable to monitor its execution. Just like any other Java heap dump, it can be opened with the [VisualVM](https://www.graalvm.org/latest/tools/visualvm/) tool. 
```

Comment:
Rewrite this paragraph to:
You can create a heap dump of a running executable to monitor its execution. Just like any other Java heap dump, it can be opened with the [VisualVM](https://www.graalvm.org/latest/tools/visualvm/) tool. To enable heap dump support, native executables must be built with the `--enable-monitoring=heapdump` option.